core/db_models/EEM_Form_Input.model.php

```diff
@@ -58,19 +57,21 @@ protected function __construct(Element $element, InputTypes $input_types, $timez
         $this->plural_item   = esc_html__('Form Inputs', 'event_espresso');
 
         $this->_tables          = [
-            'Form_Input' => new EE_Primary_Table('esp_form_input', 'FIN_ID'),
+            'Form_Input' => new EE_Primary_Table('esp_form_input', 'FIN_UUID'),
```

Comment:
i had the primary keys set incorrectly on these lines for this model as well as the form section and form submission models. Totally overlooked this because I assumed the PK was set when you you used one of the PK field types in the table declaration. 🤷🏻‍♂️ oh well